contributors/devel/api_changes.md

```diff
@@ -479,7 +508,30 @@ use our own equivalents for JSON serialization. The `pkg/api/serialization_test.
 will verify that your protobuf serialization preserves all fields - be sure to
 run it several times to ensure there are no incompletely calculated fields.
 
-## Edit json (un)marshaling code
+### Generate Clientset
+
+See [this document](generating-clientset.md) specific to generating client sets.
+
+### Generate Listers
+
+`lister-gen` is a tool to generate listers for a client. It requires a
+client generated by `client-gen`.
+
+`lister-gen` requires the `// +genclient=true` annotation on each
```

Comment:
I *think* this has to be a line above the struct with a space in-between.  Can you confirm?  If so, this seems a rather important detail.  e.g. 

```go
// +genclient=true

type MyType struct {
```

instead of 

```go
// +genclient=true
type MyType struct {
```